### DISCUSSION TITLE
Enable / Disable kernel module autoload and service autostart

### DISCUSSION THREAD

**Discussion by ghost** [runfalk/synology-wireguard#161] [Q&A]
I'm looking for how to disable the wireguard kernel module from loading at boot time. As has been mentioned in other threads, the module must be re-compiled after every DSM update / Kernel upgrade. So, if the wg module loads at boot if not at first being upgraded after one of these things occurs, bad things happen. 

Ideally, I'l like the kernel module to never load at boot and always have to be manually loaded. Via `modprobe` maybe? I'm not sure how that would happen either.

So, the 4 things I'm looking for specifically are:

1) `command` to disable autoloading the wg kernel module forever
2) `command` to disable the wg service from auto starting forever
3) `command` to manually load the wg kernel module
4) `command` to manually start the wg service

**Comment by carmatana** [comment]
Did you find the answers ?
